reposerver/repository/repository.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/TomOnTime/utfutil"
 	argoexec "github.com/argoproj/pkg/exec"
 	"github.com/ghodss/yaml"
-	"github.com/google/go-jsonnet"
+	jsonnet "github.com/google/go-jsonnet"
```

Comment:
I'm assuming this came from my vim setup running `goimports` or `go fmt`. This wasn't done intentionally.